templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -190,5 +190,136 @@
 				</div>
 			{{end}}
 		{{end}}
+
+		{{if .IssueDependenciesEnabled}}
+		<div class="ui divider"></div>
+
+		<div class="ui depending">
+			<span class="text"><strong>{{.i18n.Tr "repo.issues.dependency.title"}}</strong></span>
+			<br>
+			{{if .BlockedByDependencies}}
+			<span class="text">
+				{{if .Issue.IsPull}}
+					{{.i18n.Tr "repo.issues.dependency.issue_closing_blockedby"}}:
+				{{else}}
+					{{.i18n.Tr "repo.issues.dependency.pr_closing_blockedby"}}:
+				{{end}}
+			</span>
+			<div class="ui relaxed divided list">
+				{{range .BlockedByDependencies}}
+				<div class="item">
+					<div class="right floated content">
+						<a class="delete-dependency-button" onclick="deleteDependencyModal({{.Issue.ID}}, 'blockedBy');">
+							<i class="delete icon text red"></i>
+						</a>
+						{{if .IsClosed}}
+						<div class="ui red mini label">
+							<i class="octicon octicon-issue-closed"></i>
+						</div>
+						{{else}}
+						<div class="ui green mini label">
+							<i class="octicon octicon-issue-opened"></i>
+						</div>
+						{{end}}
+					</div>
+					<div class="ui black label">#{{.Index}}</div>
+					<a class="title has-emoji" href="{{$.RepoLink}}/issues/{{.Index}}">{{.Title}}</a>
+				</div>
+				{{end}}
+			</div>
+			{{end}}
+
+			{{if .BlockingDependencies}}
+			<div class="ui relaxed divided list">
+				<span class="text">
+					{{if .Issue.IsPull}}
+						{{.i18n.Tr "repo.issues.dependency.pr_close_blocks"}}:
+					{{else}}
+						{{.i18n.Tr "repo.issues.dependency.issue_close_blocks"}}:
```

Comment:
What about using some short strings here and add full explanation as tooltip?

Reply:
I think it is more clear the way it is.

Reply:
If you read it first time it's ok, but if you see it again, it's too much. UI should be clear as much as possible, using long sentences isn't necessary here. Using short, appropiate title would be far better. Maybe simple `Blocked by` and `Is blocking`.

Reply:
I think `Blocked by` would be better as `Depends on`

Reply:
Done.